tests/host_tools/memory.py

```diff
@@ -94,6 +96,12 @@ def run(self):
         while not self._should_stop:
             mem_total = 0
             try:
+                if (
+                    platform.machine() == "x86_64"
+                    and (self.guest_mem_mib * 1024) > self.X86_MEMORY_GAP_START
+                ):
+                    # Remove once GitHub issue #3349 is solved.
```

Comment:
```suggestion
                   # TODO Remove when <https://github.com/firecracker-microvm/firecracker/issues/3349> is closed.
```

Reply:
Applied ur suggestion.
Sounds good to add a reference.